pkg/operation/care/checker.go

```diff
@@ -548,6 +550,18 @@ var managedResourcesShoot = sets.NewString(
 	metricsserver.ManagedResourceName,
 )
 
+var managedResourcesSeed = sets.NewString(
+	etcd.Druid,
+	seedadmissioncontroller.Name,
+	common.ManagedResourceDependencyWatchdogEndpoint,
+	common.ManagedResourceDependencyWatchdogProbe,
+)
```

Comment:
There are more, why didn't you add them?

`global-network-policies`
`cluster-identity`
`cluster-autoscaler`
...